data-serving/data-service/test/controllers/case.test.ts

```diff
@@ -69,10 +77,16 @@ beforeAll(async () => {
     mockLocationServer.listen();
     mongoServer = new MongoMemoryServer();
     await createAgeBuckets();
-    curator = await User.create({ email: curatorUserEmail });
+    curator = await User.create(
+        { 
+            name: curatorName,
+            email: curatorUserEmail,
+            roles: [Role.Curator]
+        }
+    );
     minimalDay0CaseData = {
         ...minimalDay0CaseData,
-        ...{ curators: { createdBy: curator._id } },
+        ...{ curators: { createdBy: curatorMetadata.curator } },
```

Comment:
I updated it (and also one in the full case) from:
```
...{ curators: curatorMetadata.curator },
```
to:
```
...{ curators: { createdBy: curatorMetadata.curator } },
```
this triggered the hidden if coupling described below.